assets/js/src/marketing_optin_block/block.tsx

```diff
@@ -0,0 +1,50 @@
+/* eslint-disable react/react-in-jsx-scope */
```

Comment:
It would be nice to get rid of this statement, can we instead import react?

Reply:
This would cause `@wordpress/dependency-extraction-webpack-plugin` to include `react` as a dependency when registering/enqueueing the block script. It's not neccessary.